src/content/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor.mdx

```diff
@@ -5,32 +5,35 @@ title: New Relic Azure Monitor integration
 import infrastructureAzureMonitorSelect from 'images/azure-monitor.png'
 
 <Callout title="Public preview">
-This feature is currently a public preview.
+This feature is currently in public preview.
 </Callout>
 
-
 Monitor and report data about your Microsoft Azure services to New Relic with the New Relic Azure Monitor integration.
 
-Our Azure Monitor integartion improves upon our past Azure solutions;
-- Coverage: New metrics will be available the moment Azure adds it to their Azure Monitor API endpoints, including data from new Azure services.
-- Always up-to-date: Any improvement in existing metrics will show up in New Relic automatically.
-- Faster metrics: The new integration enables customers to define up to 1 polling interval reducing the average time-to-ingest for most Azure Monitor namespaces.
+Our Azure Monitor integration improves upon our past Azure solutions: 
+
+* Coverage: New metrics will be available the moment Azure adds it to their Azure Monitor API endpoints, including data from new Azure services.
+* Always up-to-date: Any improvement in existing metrics will show up in New Relic automatically.
+* Faster metrics: The new integration enables customers to define up to 1 polling interval reducing the average time-to-ingest for most Azure Monitor namespaces.
+
+## The Azure Native New Relic Service [#azure-native]
 
-## Features
+Our Azure Monitor integration is the integration that powers the Azure Native New Relic Service. This service lets you set up and configure New Relic directly from Azure, and store your New Relic data in Azure. [Learn more about the Azure Native New Relic Service.](/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native/)
```

Comment:
I think you can simplify and reduce redundancy in the first sentence by removing that second "integration".

Reply:
I'd remove that comma in the second sentence